src/ContextAwareInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Dhii\Output;
+
+/**
+ * Something that is aware of a context.
+ *
+ * @since [*next-version*]
+ */
+interface ContextAwareInterface
+{
+    /**
+     * Retrieves the context.
+     *
+     * @since [*next-version*]
+     *
+     * @return mixed
```

Comment:
"The context". Let's try to never leave the type docs without some description. Except maybe if it's `@return $this` :D